### PR TITLE
Use normalize_whitespace filter for meta description

### PR DESCRIPTION
**Pull request by pathawks**
Taking advantage of jekyll/jekyll#4917 to solve jekyll/jekyll#5066

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
   <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
-  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description | escape }}{% endif %}">
+  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | normalize_whitespace | truncate: 160 }}{% else %}{{ site.description | escape | normalize_whitespace }}{% endif %}">
```
Comment (pathawks) [inline _includes/head.html]: Is there some reason for the two different filter chains? Why not:

```text
<meta name="description" content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 }}">
```

Comment (parkr) [inline _includes/head.html]: `site.description` is also run through `escape`. Other than that, I see no big reason they should be separate. `page.excerpt` could be run through `escape` as well.

**Comment by pathawks** [comment]
Updated to get rid of the `if` :+1:

**Comment by parkr** [comment]
LGTM.

**Comment by parkr** [comment]
LGTM.

**Comment by parkr** [comment]
@jekyllbot: merge

**Pull request merged**